src/_pytest/fixtures.py

```diff
@@ -1728,6 +1728,20 @@ def _register_fixture(
         )
 
         faclist = self._arg2fixturedefs.setdefault(name, [])
+
+        by_plugin = fixture_def.baseid == ""
```

Comment:
If the fixture comes from the `conftest.py` file in the root path, it will bypass this check.